frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/personnel-type-to-german-abbreviation.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { PersonnelType } from 'digital-fuesim-manv-shared';
+
+const personnelTypeToGermanAbbreviationDictionary: {
```

Comment:
I think this should be called `personnelTypeToAbbreviationDictionary` because we don't really care if it is German and anyone who might want to internationalize the project would have to refactor those identifier names.

Reply:
See below